klippy/toolhead.py

```diff
@@ -561,7 +568,7 @@ def get_status(self, eventtime):
                      'position': self.Coord(*self.commanded_pos),
                      'max_velocity': self.max_velocity,
                      'max_accel': self.max_accel,
-                     'max_accel_to_decel': self.requested_accel_to_decel,
+                     'minimum_cruise_ratio': self.min_cruise_ratio,
                      'square_corner_velocity': self.square_corner_velocity})
```

Comment:
I think including "'max_accel_to_decel': self.max_accel_to_decel" still in response may be good for backward compatibility of older frontends which cannot be easily upgraded.

Reply:
I think you shouldn't update Klipper if the frontend you are using cannot be updated easily. What makes the frontend update difficult? Some locked down chinese os/hardware?

Reply:
Thank you. I agree with your comment. The difficulty of upgrading is due to my environment.

The reason for my comment was that older frontends displaying incorrect values in the max_atd box without errors. It made me think that it might be better to return the correct value from internal max_atd for a while. But now I understand that is a rare situation in reality.